### PR TITLE
Update eep-0049 with decisions from OTP Technical Board

### PR DESCRIPTION
**Pull request by bjorng**
Introduce `maybe` as keyword to start the construct and `<~' as matching operator.

### CODE REVIEW

File: eeps/eep-0049.md [hunk 1]
```diff
@@ -160,9 +157,9 @@ construct:
 
     commit_write(OpaqueData) ->
         begin
```
Comment (zuiderkwast) [inline eeps/eep-0049.md:159]:
```suggestion
        maybe
```

File: eeps/eep-0049.md [hunk 2]
```diff
@@ -171,9 +168,9 @@ Or, to protect against `disk_log` calls returning something else than `ok |
...
```
Comment (zuiderkwast) [inline eeps/eep-0049.md:170]:
```suggestion
        maybe
```

File: eeps/eep-0049.md [hunk 3]
```diff
@@ -221,24 +218,25 @@ as:
 
     pre_check(Action, User, Context, ExternalThingy) ->
         begin
```
Comment (zuiderkwast) [inline eeps/eep-0049.md:220]:
```suggestion
        maybe
```

File: eeps/eep-0049.md [hunk 4]
```diff
@@ -1399,38 +1402,30 @@ is chosen following Erlang/OTP standards:
...
-compatibility concern to be had.
-
-The need for the `<-` operator to be used in a new context means code built
-with support for the new expressions won't be portable to older Erlang
-releases. However, if all the code is rewritten _after_ the AST and reuses
-existing core Erlang components, built BEAM artifacts should work on older
-versions fine. This is, however, not a supported use case by the OTP team.
+The plan is to introduce this new feature together with a mechanism
+that let the user activate it and other new features individually per
+module and by that the new keyword `maybe` will only be a potential
```
Comment (zuiderkwast) [inline eeps/eep-0049.md:1418]: Will there be another EEP for this mechanism?

Will a feature become activated-by-default after some number of OTP versions?

**Review by zuiderkwast — COMMENTED** [review]
I'm not sure whether I like it or not, but I found some typos. :-)

**Review by ferd — APPROVED** [review]

File: eeps/eep-0049.md (hunk 4 above)
Comment (bjorng) [reply eeps/eep-0049.md:1418]: > Will there be another EEP for this mechanism?

Yes, there is ongoing work on both an EEP and an implementation. We are still doing internal reviews of the EEP.

> Will a feature become activated-by-default after some number of OTP versions?

If a feature turns out to be successful, we will activate it by default in a future release (and there might be a possibility that the feature could still be possible to be turned off for legacy code).

**Pull request merged**